catalog/core/catalog-core-api-impl/src/main/java/ddf/catalog/data/impl/MetacardImpl.java

```diff
@@ -158,7 +158,11 @@ public MetacardImpl(Metacard metacard, MetacardType type) {
         }
         map = new HashMap<>();
         for (AttributeDescriptor attribute : metacard.getMetacardType().getAttributeDescriptors()) {
-            map.put(attribute.getName(), metacard.getAttribute(attribute.getName()));
+            Attribute metacardAttribute = metacard.getAttribute(attribute.getName());
```

Comment:
@lessarderic Reading the documentation it looks like it is already actually saying that.

> ...but will create a new metacard and clone any attributes defined by the {@link MetacardType} that exist on the given {@link Metacard}.